src/OpenTelemetry.Extensions.Enrichment/README.md

```diff
@@ -0,0 +1,11 @@
+# OpenTelemetry .NET SDK telemetry enrichment framework
```

Comment:
It will be great to include empty CHAGNELOG.md file.

Reply:
Added, thanks